VargasMartinezJose/Dis45/HackerRank/ejerciciojava.java/JavaIfElse.java

```diff
@@ -0,0 +1,24 @@
+package bootcamp.java.day.two.ifelse;
+
+import java.util.Scanner;
+
+public class JavaIfElse {
+
+    private static final Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+
+        int number = scanner.nextInt();
+        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
+
+        if (number % 2 == 1 || (number % 2 == 0) & (number >= 6 & number <= 20)) {
```

Comment:
What's the difference between & and && ???
Really want to use & ?

Reply:
&& will check if "number" is even and in case it's false it won't be  necessary to check the other condition. I’ll modify it in a moment.

Reply:
Thank you!